src/frontend-samples/explode-sample/ExplodeUI.tsx

```diff
@@ -0,0 +1,224 @@
+      elementIds: ["0x20000000f01", "0x20000000ef1", "0x20000000f08", "0x200000009a2", "0x200000009a3", "0x20000000eed", "0x20000000eee", "0x20000000efb", "0x200000007fd", "0x20000000efd", "0x20000000eef", "0x20000000ef0", "0x20000000eff", "0x200000009a1", "0x20000000efe", "0x20000000f0f", "0x20000000f10", "0x20000000f06", "0x200000009a0", "0x20000000f12", "0x20000000f11", "0x20000000f00", "0x200000009a5", "0x20000000eec", "0x20000000ee5", "0x20000000f02", "0x20000000f03", "0x20000000eea", "0x20000000f04", "0x20000000ef4", "0x20000000ef6", "0x20000000ef5", "0x20000000f05", "0x20000000ef9", "0x20000000f0a", "0x20000000ef8", "0x20000000ef7", "0x20000000ee6", "0x20000000991", "0x2000000099a", "0x20000000992", "0x20000000eeb", "0x20000000ef3", "0x20000000ef2", "0x20000000f0c", "0x20000000ee7", "0x20000000997", "0x20000000f0b", "0x20000000f14", "0x20000000f13", "0x20000000ee9", "0x200000009a4", "0x20000000998", "0x20000000ee3", "0x20000000999", "0x20000000996", "0x20000000994", "0x20000000993", "0x20000000995"],
+    },
+    {
+      name: "Table",
+      elementIds: ["0x200000009b5", "0x200000009b4", "0x200000009af", "0x200000009ae", "0x200000009b1", "0x200000009b0", "0x200000009b3", "0x200000009b2", "0x200000009ac", "0x200000009ad", "0x200000009a9", "0x200000009aa", "0x200000009ab"],
+    },
+    {
+      name: "Sunroom",
+      elementIds: ["0x20000000883", "0x20000000907", "0x20000000908", "0x20000000906", "0x20000000904", "0x200000001c6", "0x200000001c5", "0x200000001c7", "0x200000001c9", "0x20000000885", "0x20000000886", "0x20000000887", "0x20000000888", "0x20000000905", "0x20000000884", "0x20000000903", "0x200000001ca", "0x200000008ff", "0x200000001ce", "0x200000001cf", "0x200000008fe", "0x20000000900", "0x200000001cd", "0x20000000902", "0x20000000901", "0x200000001cb", "0x200000001cc", "0x2000000088b", "0x2000000088f", "0x2000000088e", "0x2000000088d", "0x20000000889", "0x20000000f0b", "0x20000000ee7", "0x20000000ef2", "0x20000000ee8", "0x20000000126", "0x20000000125", "0x20000000890", "0x2000000018c", "0x2000000088c", "0x2000000088a", "0x200000001b1", "0x200000001c8"],
+    },
+  ];
+
+  constructor(props: SampleProps) {
+    super(props);
+    this.state = {
+      isAnimated: false,
+      isInit: true,
+      object: this._objects.find((o) => o.name === "Table")!,
+      explodeFactor: (ExplodeApp.explodeAttributes.min + ExplodeApp.explodeAttributes.max) / 2,
+      emphasize: EmphasizeType.None, // This will be changed to Isolate before the exploded view effect is applied.
+    };
+  }
+
+  /** Kicks off the exploded view effect. */
+  public explode() {
+    const vp = this.state.viewport;
+    if (!vp) return;
+    if (this.state.isInit) {
+      this.setState({ isInit: false, emphasize: EmphasizeType.Isolate });
+    }
+    ExplodeApp.refSetData(vp, this.state.object.name, this.state.object.elementIds, this.state.explodeFactor);
+  }
+
+  /** Creates and starts an animator in the viewport. */
+  public createAnimator(): Animator {
```

Comment:
Might be worth adding this to the list of "purposes of this sample" in README.

Reply:
Referenced in "purpose" of README.